docs/content/tutorials/building-comfort/_index.md

```diff
@@ -235,22 +235,21 @@ The easiest way to follow along with this tutorial is to launch a Github
   application within your browser without setting up anything on your own
```

Comment:
In the Curbside tutorial, there are a few screenshots that guide the users through the process. We should probably do the same for the building comfort. 

I think the screenshots will be particularly useful for those who have never used codespace before